helm_deploy/whereabouts-api/values.yaml

```diff
@@ -8,7 +8,7 @@ ingress:
   # live-1 is blue, live is green
   contextColour: green
   annotations:
-    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
```

Comment:
Doesn't this need to be kept at 100?

Reply:
the old one is set to 0 (which is a valid value) and the new to 100 as per the guide